R-package/R/xgb.Booster.R

```diff
@@ -751,28 +642,292 @@ xgb.config <- function(object) {
   for (i in seq_along(p)) {
     .Call(XGBoosterSetParam_R, handle, names(p[i]), p[[i]])
   }
-  if (is(object, 'xgb.Booster') && !is.null(object$raw)) {
-    object$raw <- xgb.serialize(object$handle)
+  return(object)
+}
+
+#' @rdname getinfo
+#' @export
+getinfo.xgb.Booster <- function(object, name) {
```

Comment:
Is there any function in R that can be integrated with this info? Like `colnames`.

Reply:
I don't know of any. I don't think `colnames` would be appropriate, since the booster doesn't hold the data and doesn't have dimensions when used calling `dim` on it.

Moreover, if one can slice it to subset boosting rounds, I don't think it'd be logical to then have the column names refer to something entirely different.

Perhaps @mayer79 or @jameslamb could give some suggestions here.

Reply:
Actually, on second thought, there is `stats::variable.names`. Will add it as an option here.

Reply:
Added `variable.names` method for `xgb.Booster`.

Reply:
Nice!